sdf.md

```diff
@@ -1449,7 +1447,10 @@ consumption there is no conflict with the intended goal.
 
 ## sdfThing
 
-An sdfThing is a set of declarations and qualities that may be part of a more complex model. For example, the object declarations that make up the definition of a single socket of an outlet strip could be encapsulated in an sdfThing, and the socket-thing itself could be used in a declaration in the sdfThing definition for the outlet strip
+An sdfThing is a set of declarations and qualities that may be part of
+a more complex model. For example, the object declarations that make
```

Comment:
Should "object" be changed to "sdfObject" or "Object" here?

Reply:
Good point.  Fixed in commit below.